sync_diff_inspector/config.go

```diff
@@ -61,6 +61,8 @@ type CheckTables struct {
 
 	// table list
 	Tables []string `toml:"tables" json:"tables"`
+
+	ExcludeTables []string `toml:"exclude_tables" json:"exclude_tables"`
```

Comment:
we often use `-` in field tags, so how about renaming `exclude_tables` to `exclude-tables`? If so, do not forget to update the one in _config.toml_.

Reply:
sure, I'll change it. thanks.

Reply:
Finished.